core/src/main/java/org/bitcoinj/core/NetworkParameters.java

```diff
@@ -77,6 +77,7 @@
     protected int addressHeader;
     protected int p2shHeader;
     protected int dumpedPrivateKeyHeader;
+    protected String segwitAddressHrp;
```

Comment:
Could we rename this to `addressPrefix`?

Reply:
Hmm, is this "bc" string planned to be used for more than the HRP of segwit addresses?

Reply:
Only for HRP. I think addressPrefix is more descriptive.

Reply:
BIP173 uses the term HRP or Human Readable Part. It doesn't use Prefix. Would you prefer `segwitAddressHumanReadablePart`?

Reply:
What about addressHrp, then?

Reply:
Ah, so it's the "segwit" that you want to get rid of? But legacy addresses have no HRP.

Reply:
My comments are mainly to make the library abstract and alt-chain friendly. For example BitcoinCash refers this as "addressPrefix".